core/src/labels/labelMesh.cpp

```diff
@@ -1,8 +1,14 @@
 #include "labels/labelMesh.h"
 #include "labels/label.h"
+#include "shaderProgram.h"
 
 namespace Tangram {
 
+GLuint s_quadIndexBuffer = 0;
+int s_quadGeneration = -1;
+
+const size_t maxVertices = 1 << 14;
```

Comment:
Also it seems more clear to me to directly expose `16384` as the max vertices value here.